### ISSUE TITLE
Use releases

### ISSUE THREAD

**Issue by thompol** [w2c/ispconfig3_roundcube#119]
I think it would be good to start using versions for this plugin, so users can be notified of new releases. What do you think?

**Comment by thompol** [comment]
@w2c are you still maintaining the plugins?

**Comment by Churro** [comment]
> I think it would be good to start using versions for this plugin, so users can be notified of new releases. What do you think?

Agreed... but before that I'd favor to integrate the changes that came with ISPconfig 3.2, e.g., regarding rspamd.
